pages/crowdinintegration.md

```diff
@@ -1,123 +1,109 @@
 [Google Doc](https://docs.google.com/document/d/1gsDmBSq5Tu1G_Ei1CfTRUuvvwpAWUPhHYr2UYESwISI/edit?usp=sharing)
 # Crowdin Integration
-##Objectives:
+## Objectives:
 There are two objectives for this assignment:
+
 * 1- Provide exactly one screenshot for the strings in BeLL app, and tag it.
```

Comment:
why are you adding dash `-` after a number if the number is automagically formatted as numbering? You can try to delete the dash and you will have better number formatting. You also add an asterisk `*` before the number, I think better use number only formatting.